pkg/operators/ebpf/extrainfo.go

```diff
@@ -109,7 +121,153 @@ func (i *ebpfInstance) addExtraInfo(gadgetCtx operators.GadgetContext) error {
 		Content:     []byte(variablesJson),
 	}
 
+	// add mermaid graph data
+	flowchartGraph, err := i.generateFlowchartMermaidGraph()
+	if err != nil {
+		return fmt.Errorf("generating mermaid graph: %w", err)
+	}
+
+	ebpfInfo.Data["ebpf.mermaid.flowchart"] = &api.GadgetInspectAddendum{
+		ContentType: "text/mermaid",
+		Content:     []byte(flowchartGraph),
+	}
+
 	gadgetCtx.SetVar("extraInfo.ebpf", ebpfInfo)
 
 	return nil
 }
+
+func (i *ebpfInstance) generateFlowchartMermaidGraph() (string, error) {
+	tree := &Node{
```

Comment:
This variable is not used for anything. It was useful in the original code for interacting with [jstree](https://www.jstree.com/) in a website, but it is not relevant here, so all the code about `Node` and `AppendNode` can be removed.